pkgs/top-level/all-packages.nix

```diff
@@ -29076,7 +29076,14 @@ with pkgs;
 
   pulseaudio-dlna = callPackage ../applications/audio/pulseaudio-dlna { };
 
-  pulseview = libsForQt514.callPackage ../applications/science/electronics/pulseview { };
+  pulseview = libsForQt514.callPackage ../applications/science/electronics/pulseview {
+    # use the same stdenv as libsForQt415 to fix build
```

Comment:
```suggestion
    # use the same stdenv as libsForQt514 to fix build
```

Reply:
Thanks for spotting. (I fixed it but the review widget still thinks the old code exists?)